src/encoder.rs

```diff
@@ -2281,12 +2285,14 @@ fn encode_partition_bottomup(seq: &Sequence, fi: &FrameInvariants, fs: &mut Fram
     let mut subsize: BlockSize;
 
     // Code the whole block
+    // TODO(yushin): Try move PARTITION_NONE to below partition loop
```

Comment:
As separate patch, I plan to try include PARTITION_NONE case in the partition loop below.